src/react/types/types.ts

```diff
@@ -81,9 +81,11 @@ export interface ObservableQueryFields<
   TVariables extends OperationVariables,
 > {
   /** {@inheritDoc @apollo/client!QueryResultDocumentation#startPolling:member} */
-  startPolling(pollInterval: number): void;
+  // startPolling(pollInterval: number): void;
+  startPolling: ObservableQuery<TData, TVariables>["startPolling"];
```

Comment:
Alternative options:

Arrow function
```suggestion
  startPolling: (pollInterval: number) => void;
```

`this: void,`
```suggestion
  startPolling(this: void, pollInterval: number): void;
```